R/data.table.R

```diff
@@ -484,7 +484,7 @@ replace_dot_alias = function(e) {
           if (identical(nomatch, 0L) && allLen1) irows = irows[irows != 0L]
         } else {
           if (length(xo) && missing(on))
-            stop("Internal error. Cannot by=.EACHI when joining to a secondary key, yet") # nocov
+            stop("Internal error. Cannot by=.EACHI when joining to an index, yet") # nocov
```

Comment:
also checked w `grep -r "secondary ind" R` to be sure there are no other external references to deprecated `secondary index` term